web/src/AnalyticsNudge.tsx

```diff
@@ -2,6 +2,19 @@ import React, { Component } from "react"
 import "./AnalyticsNudge.scss"
 
 class AnalyticsNudge extends Component {
+  static analyticsOpt(optIn: boolean) {
```

Comment:
Oooh static, fancy!

Reply:
...is it? mostly Prettier just yelled at me until I did it.

Reply:
Interesting, do you mean TypeScript? I've never seen that before. It's technically correct since `analyticsOpt` isn't doing anything with the instance of the class. I just didn't even realize that TypeScript had static!